lnbits/core/templates/core/extensions.html

```diff
@@ -239,8 +235,7 @@
             v-if="extension.isInstalled && extension.installedRelease"
             class="float-right"
           >
-            <q-badge>
-              {% raw %}{{ extension.installedRelease.version }}{% endraw %}
+            <q-badge v-text="extension.installedRelease.version">
```

Comment:
I think this one will nuke the `q-tooltip` inside

Reply:
moved to span